### ISSUE TITLE
Implement Real-time Location Updates on Google Maps in UI

### ISSUE THREAD

**Issue by tanmayrauth** [CodingIndia/travel-safe#51]
A maps UI which will update and show the relatime tracking of the user who has shared the location.

This UI will open up when 
1. user clicks on notification of location shared
2. user clicks on list of currently share locations. ( I.e locations share to user )
3. Direct link sharing. (If someone shared it using link of travel safe then this UI should open up)

**Comment by CapTen101** [comment]
Hi there! @tanmayrauth 
Do we have to create a UI first and then proceed to implement the feature or do we have it already and only it's needed to be coded?

**Comment by tanmayrauth** [comment]
yeah UI need to be developed then the backed functionalities mentioned above, are you interested in working this?

**Comment by CapTen101** [comment]
@tanmayrauth I've just started with flutter (a week ago) so maybe It would take me some time to implement features related to Maps SDK in Flutter.
So I would say don't assign it to me as of now since I don't want this issue to be blocked for other users. I'll get back to the issue in two weeks or so when I feel confident enough to tackle it. 😃

**Comment by tanmayrauth** [comment]
No worries. Keep learning raisei in slack channel if you face any doubts.

**Comment by CapTen101** [comment]
I'm not adding on the slack channel actually. I couldn't find any such in the Contributing.md file.
Can you guide me through it? 🙂

**Comment by TahaAli07** [comment]
@CapTen101 I have invited you to join the slack group. Please check your email - trohilla@ch.iitr.ac.in

**Comment by CapTen101** [comment]
Thanks! I'll join the slack right away

**Comment by TahaAli07** [comment]
@CapTen101 Can you send me the email you used to register for gssoc ? I will then add you to the group.

**Comment by CapTen101** [comment]
Pardon for not mentioning it before, but i didn't registered for gssoc. I just came across this repository as i was getting started in flutter and wanted to contribute to some projects which are in their beginning phase. If I don't fulfill the prerequisite for getting added in the slack then no probs. I'll still be willing to contribute

**Comment by TahaAli07** [comment]
Oh Okay @CapTen101 , unfortunately the GS Organizers don't allow non gssoc participants. But it is open source , so you should be able to contribute , can he contribute to the project @tanmayrauth  ??

**Comment by tanmayrauth** [comment]
Yes

**Comment by tanmayrauth** [comment]
@sanxy Would you like to pick this issue?

**Comment by sanxy** [comment]
Yes i will like to work on this

**Comment by tanmayrauth** [comment]
Take an example of live shared location and try implementing it

**Comment by sanxy** [comment]
![Screenshot_20200503-185039](https://user-images.githubusercontent.com/15203737/80921729-fcc6e180-8d6f-11ea-9650-aed1e933b538.jpg)
Map UI is blank

**Comment by ankitshaw** [comment]
Hi @sanxy I am not getting you, what is the issue? How can I help you?

**Comment by sanxy** [comment]
The map UI is blank which is not supposed to be, the notification for shared location is not completed yet. What I should use to implement the features required are not completed yet so I think I will take up another issue till the features are done and merged.

**Comment by ankitshaw** [comment]
Hi @tanmayrauth Please help him with this

**Comment by bharatmk2567** [comment]
@sanxy you will need google maps API key for this

you can check docs:
https://developers.google.com/maps/documentation

these are some links that might help you:
https://pub.dev/packages/google_maps_flutter
https://codelabs.developers.google.com/codelabs/google-maps-in-flutter/#0

**Comment by sanxy** [comment]
@bharatmk257 just seeing this, thanks i got the map working, seems the API key in the project isn't working.

**Comment by tanmayrauth** [comment]
@sanxy is it working for you now? I hope you understand the usecase for it.
It should just take input for real time location and show up on maps. Which should then be easily integrated with other location share module eg location share list.

**Comment by sanxy** [comment]
Yes working on that, but i think the group creation and adding of members should be done before the realtime location should be implemented and tested. If i implement it now, there is no way to know if there are issues or not.

**Comment by sanxy** [comment]
I will come back to this, let me work on the create group UI
